core/tests/test_utils.py

```diff
@@ -670,6 +789,7 @@ def tearDown(self):
         self.testbed.deactivate()
 
     def _get_all_queue_names(self):
+        """Returns all the queue names."""
```

Comment:
Add `Returns:` docstring (including type info), for consistency.

Reply:
Done!